docs/source/concepts/Sample.rst

```diff
@@ -73,7 +75,7 @@ Sample Properties
 Multiple Samples
 ################
 
-The ``Sample()`` method actually returns a collection, however if you do not specify which sample you are after you will get he first member of the collection. So 
+The ``Sample()`` method actually returns a collection, however if you do not specify which sample you are after you will get the first member of the collection.
```

Comment:
I think there might be a typo in this sentence? 'however if you do not specify which sample you are *using* ~~after~~ you will get the first member of the collection'

Reply:
Could be better phrased for non native english speakers. it doesn't read very easy. to say "what you are after" meaning "what you want"